src/crdt.rs

```diff
@@ -2059,7 +2072,11 @@ mod tests {
         assert_eq!(node.sockets.gossip.local_addr().unwrap().ip(), ip);
         assert_eq!(node.sockets.replicate.local_addr().unwrap().ip(), ip);
         assert_eq!(node.sockets.requests.local_addr().unwrap().ip(), ip);
-        assert_eq!(node.sockets.transaction.local_addr().unwrap().ip(), ip);
+        let num_tx_sockets = node.sockets.transaction.len();
+        assert!(num_tx_sockets > 1);
+        for i in 0..num_tx_sockets {
```

Comment:
How about:

```rust
for tx_socket in node.sockets.transaction {
    assert_eq!(tx_socket.local_addr().unwrap().ip(), ip);
}
```

for this one and each of the for loops below?

Reply:
Done